src/solve.jl

```diff
@@ -26,6 +26,22 @@ end
             (a[1,1]*a[2,2] - a[1,2]*a[2,1])*b[3]) / d )
 end
 
+for Sa in 2:3  # not needed for Sa = 1;
```

Comment:
In the rest of the code `Sa` is typically a `Tuple`. For consistency and ease of reading, could you change this to `for Sa in [(2, 2), (3, 3)]` and update the rest of the code accordingly?

Reply:
Addressed in the latest version.